.github/workflows/test-devcontainer.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test DevContainer Image
```

Comment:
@TAM360 

Thank you a lot for the PR. 🎉 
That is an amazing work.

Following some nits

- We use the emojis and PR title to generate the docs. So, the 🌱 is used for changes that does not impact the end users and will not end up in the release notes. In this case we are adding here a new feature so that would be ✨ . I will update the title just to share how usually we do that and I hope that you do not mind
- We need to have only 1 commit, so can you please squash all (git rebase -i HEAD~4 and the add s as prefix of the commits for you are able to have all squashed in 1 commit)
- By last, we should not have any commented code before we get this one merged.

Reply:
> - We need to have only 1 commit, so can you please squash all (git rebase -i HEAD~4 and the add s as prefix of the commits for you are able to have all squashed in 1 commit).
> - By last, we should not have any commented code before we get this one merged.

I'll remove all of the commented code squash all the commits once I my GH Action is ready.

> I will update the title just to share how usually we do that and I hope that you do not mind
Please go ahead.